superset-frontend/packages/superset-ui-chart-controls/src/components/labelUtils.tsx

```diff
@@ -23,21 +23,18 @@ import { ColumnMeta, Metric } from '@superset-ui/chart-controls';
 
 const TooltipSectionWrapper = styled.div`
   ${({ theme }) => css`
-    display: flex;
-    flex-direction: column;
+    display: -webkit-box;
+    -webkit-line-clamp: 40;
```

Comment:
I haven't looked at all of 'em, but you might be able to live without the webkit prefix these days:
https://caniuse.com/mdn-css_properties_line-clamp

Reply:
By the way, some sources claim that `line-clamp` works and then... provide an example with `-webkit-line-clamp`